frontend/src/libraries/explorer-wamp/details.ts

```diff
@@ -10,7 +10,7 @@ export interface Details {
 }
 
 export default class DetailsApi extends ExplorerApi {
-  async getDetails(): Promise<Details> {
+  async getDetails(): Promise<Details | undefined> {
```

Comment:
Can you try a question mark operator here?

`Promise<Details?>`

Reply:
<img width="427" alt="image" src="https://user-images.githubusercontent.com/33007476/79269065-b22d0600-7e50-11ea-820e-3b9288ace200.png">